README.md

```diff
@@ -1,2 +1,6 @@
 # hello-world
 Building a test repository
+
+I am hoping to use GitHub for a variety of work related to experimental and computational cosmology. Let me know if you have any cool projects I should be aware of!
+
+For reference, I currently work on the Atacama Cosmology Telescope, Simons Observatory, and CCAT-prime projects as a graduate student in physics.
```

Comment:
This is a good list. I could include where I go to grad school if I wanted to be more specific, but I will leave that for now.

Reply:
👍 Sounds good to me.